apps/meteor/client/components/UserAutoCompleteMultiple/UserAutoCompleteMultipleFederated.tsx

```diff
@@ -24,7 +24,7 @@ type UserAutoCompleteOptions = {
 	[k: string]: UserAutoCompleteOptionType;
 };
 
-const matrixRegex = new RegExp('(.*:.*)');
+const matrixRegex = new RegExp('@(.*:.*)');
```

Comment:
@RocketChat/frontend I've added a `@`  here since it recognizes usernames without a `@` as a valid external username when it must strictly contain a `@`.